test/jot_test.gleam

```diff
@@ -1,29 +1,31 @@
-import gleam/io
 import gleam/list
-import gleam/string
-import gleeunit
+import gleam/int
 import jot
 import jot_test/support
+import testbldr
+import testbldr/pieces as testpieces
 
 pub fn main() {
-  gleeunit.main()
-}
-
-pub fn integration_test() {
-  let tests = support.load_example_test_cases()
-  use testcase <- list.each(tests)
-
-  let result = jot.to_html(testcase.djot)
-  case result == testcase.html {
-    True -> io.print_error(".")
-    False -> {
-      io.print_error("F")
-      io.print_error("\n\nTest failed: " <> testcase.file)
-      io.print_error("\n\nInput:\n" <> testcase.djot)
-      io.print_error("\nExpected:\n" <> string.inspect(testcase.html))
-      io.print_error("\nActual:\n" <> string.inspect(result))
-      io.print_error("\n")
-      panic as "integration test failed"
+  testbldr.demonstrate(
+    that: "Implemented cases pass",
+    with: {
+      use tests <- list.map(support.load_example_test_cases())
+      use i, tst <- list.index_map(tests)
+      case tst.html == jot.to_html(tst.djot) {
+        True -> {
+          use <- testbldr.named(".")
+          testpieces.Silent
```

Comment:
Why would I want to make some tests print and others not? I think I would only ever want to turn verbose printing on and off for the entire suite at once.

Reply:
🤷‍♂️ I *almost* added a little configuration toggle like that but then thought this would be more flexible. 
Maybe useful for custom impl of ignore functionality, maybe for printing specific test groups w/cli args, that sort of thing. I'll probably add verbose printing functionality based around it somehow, really not sure yet.

Reply:
I think generally it's good to try and provide the best experience than to be the most flexible for this sort of thing. Folks don't want to spent time configuring their tests.